percona/controller/pgcluster/controller.go

```diff
@@ -78,12 +85,13 @@ func (r *PGClusterReconciler) watchSecrets() handler.Funcs {
 			labels := e.ObjectNew.GetLabels()
 			crName := labels[naming.LabelCluster]
 
-			_, ok := labels[v2beta1.LabelPMMSecret]
-			if ok {
+			if len(crName) != 0 &&
+				!reflect.DeepEqual(e.ObjectNew.(*corev1.Secret).Data, e.ObjectOld.(*corev1.Secret).Data) {
 				q.Add(reconcile.Request{NamespacedName: client.ObjectKey{
 					Namespace: e.ObjectNew.GetNamespace(),
 					Name:      crName,
 				}})
+				return
```

Comment:
do we need this line?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/443/commits/b361255b2f0d32dde928ef61b49bd1c5c2d23b86